vector/src/main/java/im/vector/activity/VectorHomeActivity.java

```diff
@@ -1159,12 +1155,8 @@ private void displayCryptoCorruption() {
                 editor.putBoolean(isFirstCryptoAlertKey, false);
                 editor.commit();
 
-                AlertDialog.Builder alertDialogBuilder = new AlertDialog.Builder(this);
-                alertDialogBuilder.setMessage(getString(R.string.e2e_need_log_in_again));
-
-                // set dialog message
-                alertDialogBuilder
-                        .setCancelable(true)
```

Comment:
`.setCancelable(true)` has disappeared. Is it expected?

Reply:
@manuroe yes, AlertDialogs are cancellable by default, so this call was not necessary